packages/api/lib/writeRecords/write-granules.js

```diff
@@ -699,8 +707,8 @@ const writeGranuleFromApi = async (
     published,
     pdrName,
     provider,
-    error = {},
-    createdAt = new Date().valueOf(),
+    error = {}, // Future -- Move this logic to calling methods
```

Comment:
Generally setting defaults here isn't a good idea in supporting PATCH behavior/DRY on this.

Reply:
(This gets solved in the 3070x3 PR)